common/config.go

```diff
@@ -34,6 +34,8 @@ type Api struct {
 	// Rest timeout in milliseconds (if omitted, defaults to DefaultRestTimeout)
 	RestTimeoutMillis int64 `yaml:"rest_timeout_millis,omitempty" json:"rest_timeout_millis,omitempty"`
 	RestRetries       int   `yaml:"rest_retries,omitempty" json:"rest_retries,omitempty"`
+	// Location of the public key.
```

Comment:
I was thinking this information should be moved out into some main place that explains how this auth works.